test/deriver/pp_default.expected.ml

```diff
@@ -34,6 +34,10 @@ type 'a _array = 'a array[@@deriving default]
 let default__array = [||]
 type tuple = (int * string * int list)[@@deriving default]
 let default_tuple = (0, "", [])
+type 'a ok = (int, 'a) result[@@deriving default]
```

Comment:
Not part of this review, but something I noticed: is this expected file not actually an `mli`?

Reply:
nope it's indeed an `.ml`, see the `let` statements!